### ISSUE TITLE
The training model is not effective

### ISSUE THREAD

**Issue by awdx551** [qinzheng93/GeoTransformer#48]
Thank you for your excellent work. When I use the LGR training model without changing the code, the result on  3DMatch (about 91.0%) was still quite different from the model you provided (92.5%)

**Comment by qinzheng93** [comment]
This could be caused by two reasons.
First, different random numbers during training can affect the convergence, especially with a batch size of 1..
Second, there is a gap between correspondence and registration. As we only supervise the correspondence, we expect to achieve better registration results with better correspondences, but it is not always the case. You can try the model with best RR on the validation set. And are the IR and the FMR consistent with the results of the pretrained model?

**Comment by awdx551** [comment]
Thank you for your reply! I did not change the random numbers for training, and I did use the model with the best RR on the validation set. As you said, IR and FMR have the same or even better effect than the pretrained model. In theory, RR should not be different from the pretrained, which is what I don't understand.

**Comment by qinzheng93** [comment]
It is somehow weird. There could still be something in the model that is not deterministic during training. I will have a look at it in the future.
